verification/curator-service/ui/src/components/SourceTable.test.tsx

```diff
@@ -5,10 +5,18 @@ import { fireEvent, render } from '@testing-library/react';
 import React from 'react';
 import SourceTable from './SourceTable';
 import axios from 'axios';
+import User from './User';
 
 jest.mock('axios');
 const mockedAxios = axios as jest.Mocked<typeof axios>;
 
+const curator: User = {
```

Comment:
Worth adding a unit test for the retrieval feature? Or a case in the cypress spec.

Reply:
yup, doing that in the PR with the UI stuff in it next